packages/webapp/src/stories/List/IconDescriptionCheckboxListItem.stories.jsx

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Are there three Storybook files even though it's one component so you can show the text variants for each one individually (rather than 9 variants under one giant file)? Just curious, not a blocker for merge or anything.

I don't know Storybook organization well enough to know if there is a more concise way to show all those combinations.

Reply:
Basically yeah, I am not sure best practice either but it seemed like a good idea to have a Story for each as they will be used different places throughout the app. 🤷 could combine it if that makes sense to people but yeah.